pkgs/misc/drivers/sc-controller/default.nix

```diff
@@ -49,11 +49,13 @@ buildPythonApplication rec {
   '';
 
   checkPhase = ''
-    PYTHONPATH=. py.test
+    runHook preCheck
+    PYTHONPATH=$PYTHONPATH:. py.test
+    runHook postCheck
   '';
 
   meta = with lib; {
-    homepage    = "https://github.com/kozec/sc-controller";
+    homepage    = "https://github.com/Ryochan7/sc-controller";
     # donations: https://www.patreon.com/kozec
```

Comment:
That should be updated/removed, too.

Reply:
The original author is still working on sc-controller (the C rewrite of it), so it's fair to keep this.
IIRC Ryochan7 suggested donations should still be directed to him.

Reply:
It is a comment anyway and we don't have such comments anywhere else.

Reply:
Well, keeping it does no harm, also removing it now makes it look like we're withdrawing support.